src/app/participant-list/participant-list.component.ts

```diff
@@ -737,6 +738,9 @@ export class ParticipantListComponent implements OnInit {
       }
     );
   }
+  selectedColumnsNotEmpty(): boolean {
+    return Object.values(this.selectedColumns).find(value => value != null && value != []) !== null;
```

Comment:
what do you check here `value != []`?

Reply:
so that it's not empty array, couldn't check with length because it didn't have type

Reply:
are you sure that such checking will work? value should be array here? if so `(value: []) => value != null && value.length > 0`